src/dmd/typesem.d

```diff
@@ -165,10 +165,17 @@ private void resolveTupleIndex(const ref Loc loc, Scope* sc, Dsymbol s, Expressi
 
 /*************************************
  * Takes an array of Identifiers and figures out if
- * it represents a Type or an Expression.
- * Output:
- *      if expression, *pe is set
- *      if type, *pt is set
+ * it represents a Type, Expression, or Dsymbol.
+ * Params:
+ *      mt = array of identifiers
+ *      loc = location for error messages
+ *      sc = context
+ *      s = symbol to start search at
+ *      scopesym = unused
```

Comment:
If this is not used, perhaps remove the parameter name.

Reply:
Perhaps, but I made enough changes for this PR.